### ISSUE TITLE
Trailing line comments in indented code block are deleted by snakefmt 0.7.0

### ISSUE THREAD

**Issue by corneliusroemer** [snakemake/snakefmt#160]
Another serious bug: In an indented `if` block, trailing line comments with `#` are deleted:

Start:
```python
if config.get("s3_dst"):

    include: "workflow/snakemake_rules/upload.smk"
    include: "workflow/snakemake_rules/trigger.smk"
    # Only include rules for trigger if uploading files since the trigger
    # rules depend on the outputs from upload.
```

After formatting

```py
if config.get("s3_dst"):

    include: "workflow/snakemake_rules/upload.smk"
    include: "workflow/snakemake_rules/trigger.smk"


# Only include rules for trigger if uploading files since the trigger
# rules depend on the outputs from upload.
```

After another formatting:
```python
if config.get("s3_dst"):

    include: "workflow/snakemake_rules/upload.smk"
    include: "workflow/snakemake_rules/trigger.smk"
```

**Comment by corneliusroemer** [comment]
Workaround: put comment at the beginning of the code block
```
if config.get("s3_dst"):

    # Only include rules for trigger if uploading files since the trigger
    # rules depend on the outputs from upload.
    include: "workflow/snakemake_rules/upload.smk"
    include: "workflow/snakemake_rules/trigger.smk"
    include: "workflow/snakemake_rules/trigger.smk"
```

**Comment by mbhall88** [comment]
This bug still exists after v0.8.0 sorry. We'll try and get to it soon

**Comment by mbhall88** [comment]
This is resolved in v0.8.0 from my testing

**Comment by corneliusroemer** [comment]
Excellent, thank you so much! This appears to be fixed in 0.8 indeed in my tests.
